cdi/embedded-jakarta/src/main/java/org/infinispan/cdi/embedded/package-info.java

```diff
@@ -0,0 +1,4 @@
+/**
+ * Version of Infinispan CDI Embedded using the JakartaEE APIs.
```

Comment:
@jabolina Let's fix these in Javadoc as well and use "Jakarta EE" instead (there are ~12 occurences).